tests/integration/double_vote.go

```diff
@@ -76,6 +76,8 @@ func (s *CCVTestSuite) TestHandleConsumerDoubleVoting() {
 		s.consumerChain.ChainID,
 	)
 
+	provSigner.GetPubKey()
```

Comment:
unnoticed statement with no effect

Reply:
I don't get the comment, could you elaborate?

Reply:
This statement with no effect was added by mistake in #1232 and unnoticed during the review.

Reply:
I don't see this statemane neither as part of the #1232 nor in latest version of  origin/feat/ics-misbehaviour-handling. So it is added in this commit (20b0e35a).
As it has not effect can yoy remove it?

Reply:
Thanks @bermuell you're 100%. I got mixed up while checking the PR diff.
It's removed.